Documentation/En/Index.xyl

```diff
@@ -103,9 +103,9 @@
   <strong>unit</strong> and <strong>functional</strong> tests. We also include
   performance tests in some libraries.</p>
   <p>Remember that each point in this test universe may imply different tools
-  and different development practices. When talking about tests, this is
-  important to have this schema in minds and to wonder where our problematic
-  locates.</p>
+  and different development practices. When talking about tests, it is
```

Comment:
Sure, but “this is” is not specific, while “it is” is specific. But nevermind, in this particular case it works.

So :+1:.